html/cross-origin-opener-policy/resources/common.js

```diff
@@ -2,12 +2,16 @@ const SAME_ORIGIN = {origin: get_host_info().HTTPS_ORIGIN, name: "SAME_ORIGIN"};
 const SAME_SITE = {origin: get_host_info().HTTPS_REMOTE_ORIGIN, name: "SAME_SITE"};
 const CROSS_ORIGIN = {origin: get_host_info().HTTPS_NOTSAMESITE_ORIGIN, name: "CROSS_ORIGIN"}
 
-function url_test(t, url, channelName, hasOpener) {
+function url_test(t, url, channelName, hasOpener, openerDOMAccess) {
```

Comment:
Maybe this should be moved within a dedicated PR and update all the tests? Or is there a reason to add this specifically here?
Do we expect this to be different to hasOpener? When?

Reply:
I think we should add this check to all tests, but as a follow-up.

It is different from `hasOpener` because opener can exist when there's no COOP and the opener and openee are not same-origin. Then DOM access is not allowed due to the same-origin policy.